.github/workflows/pull-request-comments.yml

```
@@ -163,3 +163,46 @@ jobs:
             `;
 
             github.rest.issues.createComment( commentInfo );
+
+  # Leaves a comment on a pull request when no Trac ticket is included in the pull request description.
+  trac-ticket-check:
+    name: Comment on a pull request when no Trac ticket is included
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    if: ${{ github.repository == 'WordPress/wordpress-develop' && github.event_name == 'pull_request_target' }}
```

```suggestion
    if: ${{ github.repository == 'WordPress/wordpress-develop' && github.event_name == 'pull_request_target' && ! github.event.pull_request.draft && github.event.pull_request.state == 'open' }}
```

We should also make sure the PR is still open to prevent comments on any that have been closed. I also switched the other suggestion to a `!` comparison instead.